geometry_msgs/Point position
geometry_msgs/Vector3 velocity
float32 radius
